OpenTabletDriver.Desktop/Reflection/Metadata/PluginMetadata.cs

```diff
@@ -68,7 +68,7 @@ public class PluginMetadata
 
         public static string GetSHA256(Stream stream)
         {
-            using (var sha256 = SHA256Managed.Create())
+            using (var sha256 = System.Security.Cryptography.SHA256.Create())
```

Comment:
Really shouldn't need the full namespace because its included in an using statement
```suggestion
            using (var sha256 = SHA256.Create())
```

Reply:
It conflicts with member "SHA256"

Reply:
Good point, forgot we had our own impl